<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>908</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="Kenya"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="styleSheet">
       <string notr="true">gridline-color: rgb(136, 138, 133);</string>
      </property>
      <property name="locale">
       <locale language="English" country="Sweden"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="uniformRowHeights">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="animated">
       <bool>false</bool>
      </property>
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="headerHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Property</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>QObject</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <item>
        <property name="text">
         <string>objectName</string>
        </property>
        <property name="text">
         <string>treeWidget</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>QWidget</string>
       </property>
       <item>
        <property name="text">
         <string>enabled</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="background">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>252</red>
           <green>233</green>
           <blue>79</blue>
          </color>
         </brush>
        </property>
       </item>
       <item>
        <property name="text">
         <string>geaometry</string>
        </property>
        <property name="text">
         <string>[531*361]</string>
        </property>
        <item>
         <property name="text">
          <string>Y</string>
         </property>
         <property name="text">
          <string>120</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X</string>
         </property>
         <property name="text">
          <string>230</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Width</string>
         </property>
         <property name="text">
          <string>531</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Height</string>
         </property>
         <property name="text">
          <string>361</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>layoutDirection</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>QFrame</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>test</string>
       </property>
       <item>
        <property name="text">
         <string>editLine</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>callDialog</string>
        </property>
       </item>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTreeWidget" name="treeWidget_2">
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>1</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
       <item>
        <property name="text">
         <string>2-1</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>2-1-1</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
       </item>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>r0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>r1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>r2</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>c0</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>c1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>c2</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
